scenario_editor/src/Login/Login.js

```diff
@@ -1,54 +1,101 @@
 import React from 'react';
-import './Login.css';
-import Nav from '../Components/Nav'
-import Button from 'react-bootstrap/Button';
 import {Link} from 'react-router-dom';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+// import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
```

Comment:
These imports can all go on the same line ex: `import {Avatar, Button, Grid} from '@material-ui/core'`